src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
```

Comment:
Would be helpful to grab more information from the [Cookie object](https://developer.chrome.com/extensions/cookies#type-Cookie); session cookies are interesting, expiration dates are interesting, ...